WORKSPACE

```diff
@@ -18,6 +18,12 @@ git_repository(
 
 load("@org_pubref_rules_protobuf//protobuf:rules.bzl", "proto_repositories")
 
+git_repository(
+    name = "istio_toolbox",
+    remote = "https://github.com/istio/toolbox.git",
+    commit = "55ee1cfbb22863051a45f084404fdf9a77046fc4"
```

Comment:
please add a date and release annotation comment here (to match the others in the file).